components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -86,13 +83,6 @@ class PageWallet extends React.Component<Props, State> {
     })
   }
 
-  onModalBackupOnRestore = (key: string | MouseEvent) => {
-    if (typeof key === 'string' && key.length > 0) {
-      key = this.pullRecoveryKeyFromFile(key)
-      this.actions.recoverWallet(key)
```

Comment:
I think we can also remove `recoverWalletData()` in `brave_rewards_page.tsx`, `onRecoverWalletData()` in `rewards_actions.ts`, even `types.ON_RECOVER_WALLET_DATA` and the corresponding code in `wallet_reducer.ts`.

Reply:
Removed.